Introduction for section V (#14)
chhoumann/P6: sections/data-loading.tex

> As the purpose of the project is to create a system in which a user should be able to see the weather forecast for specific area, or more specifically the temperature forcast, we have developed a system that leverages several models to do the temparature forecasting. 

Reviewer: ```suggestion
As the purpose of the project is to create a system in which a user should be able to see the weather forecast for a specific area, or more precisely, the temperature forecast, we have developed a system that leverages several models to perform the temperature forecasting. 
```

The goal is to allow the user to interact with the system in such a way that the user can see the accuracy of the different models. 

> The user will be able to select a region for which they want to predict the temperature. The results will be shown both as a graph that will display the predicted value versus the actual value as well as the specific set of temperatures of that region for a given time period into the future. This is similar to what one may see on regular weather forecasts.

Reviewer: ```suggestion
The user is able to select a region for which they want to predict the temperature. The results are shown both as a graph that displays the predicted value versus the actual value, as well as the specific set of temperatures of that region for a given time period into the future. This is similar to what one may see on regular weather forecasts.
```

> In order to achieve this we have constructed a pipeline that preprocesses the data, trains our models, and the resulting models are then used to do the forecasting which is displayed on a web application. 

Reviewer: ```suggestion
In order to achieve, this we have constructed a pipeline that preprocesses the data, trains our models, and the resulting models are then used to perform the forecasting displayed on a web application. 
```

> Figure xx shows the architecture of the pipeline in broad terms. 

Reviewer: ```suggestion
Figure xx shows an overview of the architecture of the pipeline.
```

> Note that since the transformer model is the best performing model, which is shown in section (experiment afsnit), this is the model that we chosen to include and describe as part of the pipeline. Regardless of this fact, the other models are used into the pipeline in a similar fashion.

Reviewer: ```suggestion
Note that since the transformer model is the best performing model, which is shown in section (experiment afsnit), this is the model that we chose to include and describe as part of the pipeline. Regardless of this fact, the other models are used in the pipeline in a similar fashion.
```

\subsection{Data loading and preprocessing}

> In order to load the data we have done the following...

Reviewer: ```suggestion
In order to load the data, we have done the following...
```
